CHANGES.rst

```diff
@@ -6,6 +6,7 @@
   detected.
 - Revamped the naming convention of output files to have the original
   filename in it.
+- Enable PyPI
```

Comment:
This is not an edit under this tag of the code.  We should bump the version of master, then pull the change into develop.  We don't necessarily need to retag.

Reply:
Sorry, I don't follow.  Please dumb it down for me.  Thanks